src/EnergyPlus/IOFiles.cc

```diff
@@ -177,7 +177,7 @@ void InputFile::backspace() noexcept
 InputOutputFile &InputOutputFile::ensure_open(const std::string &caller, bool output_to_file)
 {
     if (!good()) {
-        open(output_to_file);
+        open(false, output_to_file);
```

Comment:
I am curious as to why this would compile before this change without the correct number of arguments?

Reply:
Never mind, I see it in the override function.